src/ProfiledAot/ProfiledAotMaui/App.xaml

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version = "1.0" encoding = "UTF-8" ?>
+<Application xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
```

Comment:
I'm going to try to rework these projects so it runs `dotnet new` into a folder, and the `Directory.Build.targets` sets everything up.